### DISCUSSION TITLE
Java support ?

### DISCUSSION THREAD

**Discussion by cch0** [gorules/zen#98] [General]
I have found this [issue](https://github.com/gorules/zen/issues/39) mentioning Java and was wondering if Java support is on the roadmap? I am mostly interested in using Zen engine inside the java program. 

Any insight is much appreciated.

**Comment by stefan-gorules** [comment]
Hi @cch0, thank you for your interest.

We plan to release .NET (C#) support until the end of year, and after that next language on our list is Java. No exact timelines however.

**Reply by cch0** [reply]
Thank you for the update.

**Reply by todurnatesh** [reply]
Any update on java language support ?

**Reply by ivanmiletic** [reply]
Hi, we do not have exact timelines yet
If Java ZEN Engine is urgent for you and your company we might be able to prioritize on our side.

Feel free to reach out on hi@gorules.io

**Comment by jeenadeepak** [comment]
Hi Team,
Any update on this?

**Comment by ivanmiletic** [comment]
@jeenadeepak @cch0 @todurnatesh we have released Java and Kotlin (JVM) support.

You can find more details here: https://mvnrepository.com/artifact/io.gorules
